docs/docsite/rst/dev_guide/developing_collections.rst

```diff
@@ -171,8 +170,24 @@ TBD.
 tests directory
 ----------------
 
-TBD. Expect tests for the collection itself to reside here.
+Executing sanity, integration, and unit tests is largely identical to performing these tests within Ansible's mainline code. ``ansible-test`` executes against modules implemented from the relative path ``ansible_collections/..`` For example, ``ansible-test`` may be executed against IOS modules from the default collection location of ``~/.ansible/collections/ansible_collections/cisco/ios/``. Tests may also be executed while in a custom directory, such as ``~/work/ansible_collections/cisco/ios/``, as long as it is the current path. Also, collections must not be executed from within Ansible's source tree and cannot reside within a directory comprised of a symbolic link.
```

Comment:
This is a very long line. Mind multi-lining it to follow the style of this doc?

Reply:
Updated.